src/08-accessing.js

```diff
@@ -171,7 +212,38 @@ function getFullAddress(person) {}
       numbers: [7185550921, 7185558611],
     };
  */
-function getFlatObject(person) {}
+function getFlatObject(person) {
+ let newPerson = {};
+ newPerson.name = person.name;
```

Comment:
Let's take another look at our assignments here - we have a very good start.  Also I would recommend that if we do nit _specifically have to delete an object key_ not to do it.  Modifying our data permanently is something we can never really come back from so we should avoid it if at all possible.